wiki/Modding/en.md

```diff
@@ -1,98 +1,237 @@
+Start by playing the selected difficulty in test mode.
+It is an important part of modding to play-test the difficulty as this allows you to understand whether or not certain [mapping techniques](/wiki/mapping_techniques) or patterns play well.
+You should also take note of anything doesn't feel right.
 
-**Rinse and repeat with the next difficulty** keep in mind that if you see something while in the Editor that you didn't notice while playing, point that out too. (It is recommended to have the map play at normal speed in the Editor while you watch it, just to double check things). When you're done, submit your post.
+After play-testing, watch the beatmap from the beatmap editor.
+It is also recommended to use a `100%` playback rate watching in the beatmap editor.
+In this stage of modding, modders frequently find more intricate (or complicated) problems such as combo errors, hit sound inconsistencies, and areas where hit circles or sliders are missing or misplaced.
 
-What to look for in a beatmap
------------------------------
+When a hit circle or slider needs attention, click on it to select.
+Now, copy the hit object (`Ctrl` + `C`) and go back to the post you started up.
+Then paste (`Ctrl` + `V`) the hit object you had just copied.
+This will paste the timestamp and combo number of the copied hit object.
+osu!web will automatically convert this to a link after you post the mod post so other modders (and the mapper) can easily access the hit objects you are talking about.
 
-A very helpful tool is AIMod, which can be opened under the "Files" tab or by pressing Ctrl+Shift+A. This shows the errors that the certain difficulty of a beatmap has. However, be wary that it is still just a computer, and some errors are not true errors (i.e. a spacing error, but there was supposed to be a jump).
+After pasting, add your observations or descriptions as to why you think that hit object should be changed.
+_For example, "I would change this spacing because the music does not suggest a jump here"._
+For every hit object you list in your mod post, you should always state the thing you think should be changed and why you think it should happen.
+
+Rinse and repeat with the other difficulties you want to mod.
+Keep in mind that if anything is amiss while in the editor, it will behave differently while playing.
+If this is the case, point it out as well.
+
+Once you are done, submit the your mod post.
+After submitting your mod post, remember that the mapper (or other modders) may request you to clarify parts of your mod post.
```

Comment:
would add something about reading the mapper's reply and reacting to it if necessary.

Because if the mapper implemented a change you suggested but misunderstood what you suggested then they might end up changing their map in a way that isnt intended by either side so this thing should probably recommend people to
1. Doublecheck the mapper's answer
2. Doublecheck the map itself as well after the mapper has applied their mod